packages/client/src/modals/ErrorModal.js

```diff
@@ -1,18 +1,39 @@
-import React from "react";
+import React, { useState } from "react";
 import Svg from "library/Svg";
 
 const ErrorModal = ({ error, onClose }) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
   return (
-    <div className="p-5 has-text-black has-text-left">
-      <div className="pl-5 columns is-vcentered is-multiline is-mobile border-light-bottom">
-        <Svg name="Warning" />
-        <h3 className="is-size-4 column is-half">Error</h3>
-      </div>
-      <div className="p-4 mt-5">
-        <span className="mr-2 mt-2  has-text-grey">{error}</span>
+    <div className="has-text-black has-text-left">
+      <div className="p-5 border-light-bottom">
+        <div className="has-text-black m-1">
+          Your transaction could not be completed. <br />
+          Please try again later.
+        </div>
+        {typeof error === "string" && (
```

Comment:
could use `lodash.isString` here and detect objects that could be stringified https://github.com/lodash/lodash/blob/4.0.1-npm-packages/lodash.isstring/index.js#L95

Reply:
Updated, thank you!